app/models/requests/page.rb

```diff
@@ -17,8 +17,7 @@ def requestable_by_all?
   end
 
   def item_commentable?
-    return super unless origin == 'SAL-NEWARK'
-    !barcoded_holdings?
+    origin == 'SAL-NEWARK'
```

Comment:
Should this be `origin == 'SAL-NEWARK' || super`? (also, what about making SAL-NEWARK a config parameter?)